Patch origin: upstream
Patch status: will be part of next version

https://git.gnome.org/browse/libxslt/commit/?id=22eed95c77a17a6ac51b9162ea6ea9a341b29345

From 22eed95c77a17a6ac51b9162ea6ea9a341b29345 Mon Sep 17 00:00:00 2001
From: Nick Wellnhofer <wellnhofer@aevum.de>
Date: Fri, 20 Dec 2013 17:33:43 +0100
Subject: Adjust expected test output

Adjust expected test output for recent changes in libxml2.
---
 tests/exslt/math/power.1.out | 2 +-
 tests/general/bug-180.err    | 2 --
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/tests/exslt/math/power.1.out b/tests/exslt/math/power.1.out
index 6c3f267..da9cb6d 100644
--- a/tests/exslt/math/power.1.out
+++ b/tests/exslt/math/power.1.out
@@ -2,7 +2,7 @@
 <results>
    <result>1</result>
    <result>7</result>
-   <result> 2.85311670611e+11</result>
+   <result>2.85311670611e+11</result>
    <result>256</result>
    <result>243</result>
 </results>
diff --git a/tests/general/bug-180.err b/tests/general/bug-180.err
index e45b36e..e0e568e 100644
--- a/tests/general/bug-180.err
+++ b/tests/general/bug-180.err
@@ -1,4 +1,2 @@
 runtime error: file ./bug-180.xsl line 4 element copy-of
 Variable 'xxx' has not been declared.
-XPath error : Stack usage errror
-xmlXPathCompiledEval: 1 objects left on the stack.
-- 
cgit v0.11.2

